jobs/data/repolist.txt

```diff
@@ -72,6 +72,8 @@ dotnet/roslyn branch=dev15.3-preview1 server=dotnet-ci
 dotnet/roslyn branch=dev15.3-preview1 subFolder=perf definitionScript=perf.groovy server=dotnet-ci2
 dotnet/roslyn branch=dev15.3-preview2 server=dotnet-ci
 dotnet/roslyn branch=dev15.3-preview2 subFolder=perf definitionScript=perf.groovy server=dotnet-ci2
```

Comment:
@shyamnamboodiripad Yes, we can delete everything that's dev15.* that's not dev15.0.x and dev15.2.x.